troposphere/static/js/components/modals/unsupported/modernizrTest.js

```diff
@@ -0,0 +1,69 @@
+define(function (require) {
+   var modernizr = require('lib/modernizr-latest.js'),
+       _ = require('underscore');
+
+   var features = modernizr;
+   var unsupportedFeatures = [];
+   var breakingFeatures = [];
+   var requiredFeatures = [
+        //this we know we have support
+        'cssanimations',
+        //this we know we don't have support
+        'regions',
+        'microdata',
+        'proximity',
+        'display-runin',
+        'mathml',
+        'dart',
+        //the actual tests so far we are concerned about
+        'es5',
+        'es5array',
+        'es5date',
+        'es5function',
+        'es5object',
+        'strictmode',
+        'es5string',
+        'json',
+        'es5syntax',
+        'es5undefined',
+        'ruby',
+        'svg',
+        'flexbox',
+        'inlinesvg',
+        'cssgradients',
+        'rgba',
+        'eventlistener',
+        'ellipsis'
+    ];
+
+    requiredFeatures.forEach(function(feat) {
```

Comment:
The number of lines of code to do this a bit long. Here is a solution that could be useful.

```
unsupported = _.map(_.filter(_.pairs(features), _.negate(_.last)), _.first)
breakingFeatures = _.intersection(requiredFeatures, unsupported)
unsupportedFeatures = _.intersection(requiredFeatures, unsupported)
```